templates/static/js/example.js

```diff
@@ -9,6 +28,16 @@
     }
   });
 
+  document.addEventListener('click', function (e) {
+    var isCodePenLink = e.target.classList.contains('js-codepen-link');
+
+    if (isCodePenLink) {
+      var form = e.target.closest('form');
+      e.preventDefault();
+      form.submit();
```

Comment:
You should null check the form here

Reply:
Now that the click event is being attached when the element is appended - there's a null check on line 191, if all of those values are empty, the form isn't created, and there's nothing to click.